Source/Scene/PointAttenuationOptions.js

```diff
@@ -0,0 +1,71 @@
+define([
+    '../Core/defaultValue'
+], function(
+    defaultValue
+) {
+    'use strict';
+
+    /**
+     * Options for performing point attenuation based on geometric error when rendering
+     * pointclouds using 3D Tiles.
+     *
+     * @param {Object} [options] Object with the following properties:
+     * {Boolean} [options.geometricErrorAttenuation=false] Perform point attenuation based on geometric error.
```

Comment:
Why not just call this `attenuation`?  The fact that is uses geometric error for sizing is an implementation detail and does not need to cloud the API and user's code.

Reply:
I was hoping to distinguish this from the post-process attenuation in the other branch.
I think it's also an important detail for the semi-technical user, since it works best based on the assumptions we're making for geometric error.

Reply:
What is the one in the other branch called and what is the context?

In general, make the most common option the most obvious name.  I suspect that this should be called `attenuation` and the other branch should be called `specialAttenuationForWhateverReason`.